src/pretix/base/models/discount.py

```diff
@@ -37,6 +37,12 @@
 from pretix.base.models import fields
 from pretix.base.models.base import LoggedModel
 
+ITEM_ID = 0
+SUBEVENT_ID = 1
+LINE_PRICE_GROSS = 2
+IS_ADDON_TO = 3
+VOUCHER_DISCOUNT = 4
```

Comment:
Not the point of this PR, but if we do this, maybe we should jump straight to a namedtuple? Still an improvement this way.

Reply:
done